### ISSUE TITLE
Filter for pairs, strategies and etc

### ISSUE THREAD

**Issue by ist89** [taniman/profit-trailer-enhancements#780]
As the title says. Using control F isn't exactly filtering, its more of a search than anything. Maybe someone already suggested this but i'd like to be able to filter through the Sales log (maybe Possible Buys log) by selecting the strategy used, or pairs. 

In essence, I have one buy strategy that is far rarer than all others and i'd like to be able to locate the sale, if any, quickly, rather than sifting through god knows how many sales, and possibly overlooking the pair that actually got bought using that particular strategy. 

Same way, i'd like to say... find ZIL in the possible buys log quicker, and not click through pages looking for it, (potentially useful at fixating on one pair and not losing it, especially since the bot constantly updates the log.)

Hope what i'm asking is making sense.

**Comment by taniman** [comment]
filtering through buy / sales log is already possible. Enable the BETA GUI and you will see.

**Comment by taniman** [comment]
![image](https://user-images.githubusercontent.com/6751341/105637540-7a71d680-5e44-11eb-8aa7-4bd64b0ca16f.png)
This sample filters and looks for STRATEGY TAKEPROFIT SAFETY

**Comment by ist89** [comment]
Neat! Didn't know about BETA GUI until now. But.. it's still a search, not a filter. Sure, i can type in Safety but itll just show me all basically all pairs that had that strategy, and not just the ones that actually used it. As in your example, the bot used A: Gain and not takeprofit safety. I'll once again have to sift through all of them looking for the one that did.

**Comment by taniman** [comment]
No no, this bots used safety else it would not say safety.  This IS a filter. 
If you mean you want to locate a STRATEGY that was actually true at the moment of the sale.  That's not really that easy and we would need to create extra logic just to make it possible. Like GAIN:true would search for GAIN when it was true. If that is what you mean. Otherwise I don't understand

**Comment by ist89** [comment]
Yes I guess i do mean to locate strategy that was true at the moment. It would make searching for it much easier.

**Comment by LiamAird** [comment]
You want to search from any strategy that is true? 
Can you explain it further?
